publisher-sdk/src/main/java/com/criteo/publisher/CriteoBannerView.java

```diff
@@ -78,6 +78,12 @@ public void loadAd() {
     }
   }
 
+  public void displayAd(@NonNull String signalData) {
```

Comment:
signalData does not seem legit here. Is it coming from openrtb or the server side bidding ?
I would propose adData, displayAdData, displayToken, displayData, ...

Reply:
It's terminology coming from the partner we are working with. Ok to use `displayData` instead.